packages/mobile/scripts/run_app.sh

```diff
@@ -1,26 +1,29 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-# ====================================
-# Configure and run the mobile app
-# ====================================
+# ========================================
+# Configure, build, and run the mobile app
+# ========================================
 
 # Flags:
-# -n: Name of the network to run on
 # -p: Platform (android or ios)
-# -f: Fast (skip steps not required unless network or depedencies changes)
-# -r: Hot Reload (Restore nav state on reload)
+# -n (Optional): Name of the network to run on 
+# -f (Optional): Fast (skip steps not required unless network or depedencies changes)
+# -r (Optional): Hot Reload (Restore nav state on reload, useful for UI dev-ing)
+# -b (Optional): Just configure and build, skip running 
```

Comment:
nit: to avoid the confusion, how about we use this: 'Just configure and build **sdk**, skip running' ?
given it doesn't build the app itself.

Reply:
Done